DuckDuckGo/MainViewController.swift

```diff
@@ -432,9 +437,11 @@ class MainViewController: UIViewController {
     }
 
     private func forgetAllAction() -> UIAlertAction {
-        return UIAlertAction(title: UserText.actionForgetAll, style: .destructive) { [weak self] _ in
+        let action = UIAlertAction(title: UserText.actionForgetAll, style: .destructive) { [weak self] _ in
             self?.forgetAll {}
         }
+        action.accessibilityLabel = UserText.confirm
```

Comment:
Maybe we should be a bit more explicit here, like "Confirm clearing all tabs"? I know people are *supposed* to know the context since they navigated to this action sheet, but still... :)